src/patternfly/demos/Masthead/masthead-template-application-launcher.hbs

```diff
@@ -1,4 +1,4 @@
-{{#> app-launcher app-launcher--id=(concat masthead--id '-application-launcher') id=app-launcher--id app-launcher--attribute=(concat 'id="' app-launcher--id '"') app-launcher--IsGrouped="true"}}
+{{#> app-launcher app-launcher--id=(concat masthead--id '-application-launcher') app-launcher--attribute=(concat 'id="' app-launcher--id '"') app-launcher--IsGrouped="true"}}
```

Comment:
if I'm reading this right, `app-launcher--id` should create `id={{app-launcher--id}}` so I'm guessing this is creating a dupe? Can't tell in dev tools since it will hide extra `id`'s - https://codepen.io/mcoker/pen/rNdXbXZ